src/PagesBody/ClassCompetitorsPage/ClassCompetitorsPage.js

```diff
@@ -0,0 +1,35 @@
+import ClassOrDogButton from '../../Molecules/ClassOrDogButton/ClassOrDogButton';
+import RANDOM_CONTESTS from '../../Data/Dummy-data/test-data-random-contests';
+import MainButton from '../../Atoms/MainButton/MainButton';
+import ColumnWrapper from '../../Templates/ColumnWrapper/ColumnWrapper';
+
+const ClassCompetitorsPage = () => {
+  function secondaryBtnHandler() {
+    console.log('secondary button clicked');
+  }
+  return (
+    <ColumnWrapper paddingLeftRight={1} paddingTop={0.25}>
+      {RANDOM_CONTESTS[0].obedienceClasses[0].dogs.map((dog, index) => {
```

Comment:
You shuoldnt be working on random anymore, there shuold be a "fetch" to the dummy data, getting the ones that you need